### ISSUE TITLE
Manual to use the DLLs

### ISSUE THREAD

**Issue by afoo93** [naudio/NAudio#945]
Hello,

I am planning to call the DLL that I built with the source code in LabVIEW for my FYP acoustics ASIO project but I found out that there is no manual and I found it is relatively difficult to access.

Could I get any help on this? Thanks

**Comment by markheath** [comment]
The NAudio documentation is available here: https://github.com/naudio/NAudio/tree/master/Docs
I'm afraid I don't have any experience of LabVIEW - I'd suggest you ask on their support forums for any information on how to access .NET DLLs from LabView

**Comment by afoo93** [comment]
Hi Mark,

May I know if this could be used to process live audio streaming? Instead of offline record and processing.

**Comment by markheath** [comment]
Yes, no reason why NAudio can't be used to process live audio, although it can be complicated depending on the protocols and codecs involved

**Comment by afoo93** [comment]
Hi Mark,

Thanks for the reply. Yet, I couldn’t found any ASIO input related parameter or class in the source code. Could you please guude me on finding the ASIO input?

**Comment by afoo93** [comment]
Also is there any way to specified which device to be used?

**Comment by markheath** [comment]
Am going to close this out as there are [several tutorials on the NAudio docs site covering ASIO playback and recording](https://naudio.github.io/NAudio/docs/AsioRecording.html) as well as samples in the demo apps, which hopefully provide you with the information you need
